src/main/java/eu/dissco/core/translator/service/BioCaseService.java

```diff
@@ -191,7 +191,7 @@ private void processUnit(DataSet datasets, Unit unit)
             physicalSpecimenId,
             termMapper.retrieveFromABCD(new Type(), unitAttributes),
             harmonizeAttributes(datasets, unitAttributes, physicalSpecimenIdType, organizationId),
-            unitAttributes
+            removeMultiMediaFields(unitAttributes)
```

Comment:
Why remove media fields just with BioCase?

Reply:
[DwcService line 183](https://github.com/DiSSCo/dissco-core-translator/blob/b6e476fff2520f89ab883ebbd4b22b82910cf521/src/main/java/eu/dissco/core/translator/service/DwcaService.java#L183) you define a list of enrichment services and set multiMediaObject to false without removing the terms in the same way you do here.

Reply:
Thanks for clarifying! That makes sense.